### ISSUE TITLE
Regarding the flow loss

### ISSUE THREAD

**Issue by archmelow** [RuijieZhu94/MotionGS#17]
Is it possible to backpropagate through pre-trained 4DGS model parameters with the given flow loss? I tried to implement it on 4D Gaussian Splatting, but the gradients to the deformation MLP network and the Gaussian parameters were all 0. Could you give any advice on implementing this, thanks in advance.

**Comment by RuijieZhu94** [comment]
Hi @ArchMelow, sorry for the late reply. I might have missed some issues... I suspect you've frozen or detached some parts of the model. If this issue is still unresolved, could you provide a more detailed description so I can better understand? Again, apologize for the late reply.

**Comment by RuijieZhu94** [comment]
Close this issue first. If you have any questions, please feel free to reopen it.
